prompting/validators/forward.py

```diff
@@ -122,19 +122,17 @@ async def run_step(
         self.device
     )
     for weight_i, reward_fn_i in zip(self.reward_weights, self.reward_functions):
-        reward_i, reward_i_normalized = reward_fn_i.apply(prompt, responses, name)
+        reward_i_normalized, reward_event = reward_fn_i.apply(prompt, responses, name)
```

Comment:
What if we get the reward event that contains a normalized value within it? That way the flow would be simplified around a single entity `reward_event`